internal/pkg/runtime/engine/singularity/process_linux.go

```diff
@@ -244,6 +244,11 @@ func (e *EngineOperations) StartProcess(masterConn net.Conn) error {
 						statusChan <- status
 					}
 				}
+			case syscall.SIGURG:
+				// Ignore SIGURG, which is used for non-cooperative goroutine
+				// preemption starting with Go 1.14. For more information, see
+				// https://github.com/golang/go/issues/24543.
+				sylog.Debugf("Ignoring SIGURG signal")
```

Comment:
Do we really care about this debug message as it's for golang runtime internal purpose ? And also a risk of flooding

Reply:
Wouldn't you want to log it at least once? Just to log the... "We got this signal sent to us and we're ignoring it."

Reply:
We're actually logging _all_ signals received in `StartProcess()`:

https://github.com/sylabs/singularity/blob/c4a0dc2fbcd0793d2b2bab6f23af1e639cdc71ba/internal/pkg/runtime/engine/singularity/process_linux.go#L230

So this `Debugf` simply results in twice the noise.

As we're not (to my knowledge) leveraging out-of-band data, the chances of a `SIGURG` that isn't related to non-cooperative goroutine preemption seems very low. And since we _are_ logging all signals, there is still a debug hint if ignoring this signal ever becomes an issue.

With that in mind, I'll remove the `Ignoring SIGURG signal` debug print unless there are any strong objections.